cmd/mimir/help-all.txt.tmpl

```diff
@@ -1751,6 +1751,8 @@ Usage of ./cmd/mimir/mimir:
     	Override the expected name on the server certificate.
   -query-frontend.instance-addr string
     	IP address to advertise to the querier (via scheduler) (default is auto-detected from network interfaces).
+  -query-frontend.instance-enable-ipv6
+    	Enable using a IPv6 instance address. (default false)
```

Comment:
Setting this to `true` is only required when using instance IP address auto-detection, right? (ie. if an IPv6 address is explicitly given to `-query-frontend.instance-addr`, then it doesn't matter what value this option has)

If so, could you please clarify the description and changelog entry to be clear that this is only relevant when an explicit address is not given?

Reply:
sure, although I just used the same description with other components like [compactor](https://grafana.com/docs/mimir/latest/references/configuration-parameters/#compactor).

Reply:
Done

Reply:
Ah, I didn't realise we used that description elsewhere sorry - in that case, it's probably better to be consistent.

Reply:
Do you want to me to revert the change?

Reply:
Sorry, yes please.

Reply:
done